packages/top-app-bar/Section.tsx

```diff
@@ -0,0 +1,55 @@
+// The MIT License
+//
+// Copyright (c) 2019 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as React from 'react';
+import * as classnames from 'classnames';
+import {cssClasses} from './constants';
+
+export interface SectionProps<T> extends React.HTMLProps<T> {
+  align?: 'start' | 'end';
+  className?: string;
+  tag?: string;
+};
+
+const Section: <T extends {} = HTMLElement>(props: SectionProps<T>) =>
```

Comment:
extend HTMLElement here too

Reply:
isn't it not already?

Reply:
I mean `<T extends HTMLElement = HTMLElement>`. This just forces `T` to always be a derivative of HTMLElement

Reply:
Ah gotcha. I think this should also be changed in the Dialog component I worked on last month. Should I open an issue? I probably won't get to implementing anything this week.

Reply:
Ah! Ya an issue would be great.